puppetdb_foreman.gemspec

```diff
@@ -1,10 +1,12 @@
+# frozen_string_literal: true
+
 require File.expand_path('lib/puppetdb_foreman/version', __dir__)
 require 'date'
 
 Gem::Specification.new do |s|
   s.name        = 'puppetdb_foreman'
   s.version     = PuppetdbForeman::VERSION
-  s.date        = Date.today.to_s
+  s.date        = Date.today.to_s # rubocop:disable Rails/Date
```

Comment:
You can also drop this line instead since it's the default behavior now.
```suggestion
```

And then also drop the require statement on line 4.